### DISCUSSION TITLE
[Question] Changing .override.yml in HuggingFace Deployment

### DISCUSSION THREAD

**Discussion by ByungsunAhn** [danny-avila/LibreChat#3749] [Q&A] [answered]
I'm pretty much a newbie in this area. I followed the tutorial to deploy my LibreChat on HuggingFace.

I know that the Environment Variable set to .env is controlled by “Variables and Secrets” in “HuggingFace Setting” (I think.)

My problem came up when I tried to set up the RAG_API.
I can't figure out where to modify the .override.yml file.

There are only 4 files in HuggingFace's Files: .gitattributes, Dockerfile, README.md, librechat.yaml and I don't know if this is the right place to modify it.

![image](https://github.com/user-attachments/assets/347f4209-c2a3-4fad-963f-424107b27148)

**Comment by fuegovic** [comment, marked as answer]
RAG_API isn't supported on HF, you would need to provide your own external solution to make it work

**Reply by ByungsunAhn** [reply]
Okay, I understand, thank you.
How do I modify the .override.yaml file?

**Reply by fuegovic** [reply]
There's no override file on huggingface, you can modify the dockerfile directly instead (in files in the top menu)

**Reply by ByungsunAhn** [reply]
Okay, I understand, thank you.
